src/cloudai/workloads/megatron_bridge/slurm_command_gen_strategy.py

```
@@ -247,7 +342,12 @@ def _list_or_comma_str(self, val: str | list[str] | None) -> Optional[str]:
             raise RuntimeError("Unexpected sweeps list. At this point code expects scalars only")
 
     def _build_launcher_parts(  # noqa: C901
```

Ruff (0.15.20)

[warning] Too many branches (28 > 12)

(PLR0912)